jekyll/_cci2/getting-started.adoc

```diff
@@ -146,71 +158,8 @@ You should soon have a passing pipeline. If you are happy with this configuratio
-Don’t see these options? Use the org selector in the top left corner to find the correct organization.
-{% endcapture %}
-
-{%- capture select-project -%}
-  {{ site.baseurl }}/assets/img/docs/getting-started-guide-exp/create-project.png
-{%- endcapture -%}
-
-{% include two-up.html title="1. Create a project" content=content imageURL=select-project imageAlt="Select Projects" %}
-
-
-{% capture content %}
-In the “Select your <a class="no-external-icon" href="https://circleci.com/docs/config-start/">config.yml</a> file” modal, select <b>Fast</b>, then give your project a name, and click <b>Create Project</b>.
-{% endcapture %}
-
-{%- capture select-config -%}
-  {{ site.baseurl }}/assets/img/docs/getting-started-guide-exp/select-config-gitlab.png
-{%- endcapture -%}
-
-{% include two-up.html title="2. Select a config.yml" content=content imageURL=select-config imageAlt="Choose Config" %}
-
-
-{% capture content %}
-You are now in the <a class="no-external-icon" href="https://circleci.com/docs/config-editor/#getting-started-with-the-circleci-config-editor">CircleCI config editor</a>, pre-populated with a sample <code>config.yml</code> file. You can swap out this config for an alternative by clicking <strong>Change: Hello World</strong>.
-{% endcapture %}
-
-{%- capture CCI-config-editor -%}
-  {{ site.baseurl }}/assets/img/docs/getting-started-guide-exp/change-config-gitlab.png
-{%- endcapture -%}
-
-{% include two-up.html title="3. CircleCI config editor" content=content imageURL=CCI-config-editor imageAlt="Config Editor" %}
-
-
-{% capture content %}
-Click <strong>Commit and Run</strong>. This will create a <code>.circleci/config.yml</code> file at the root of your repository on a new branch called <code>update-circleci-config</code>.
-{% endcapture %}
-
-{%- capture commit-run-gitlab -%}
-  {{ site.baseurl }}/assets/img/docs/getting-started-guide-exp/commit-run-gitlab.png
-{%- endcapture -%}
-
-{% include two-up.html title="4. Commit your config" content=content imageURL=commit-run-gitlab imageAlt="Commit a config" %}
-
-
-{% capture content %}
-You should soon have a passing pipeline. If you are happy with this configuration, merge it into your main branch, or continue to make changes.{% endcapture %}
-
-{%- capture congrats-first-green-pipeline -%}
-  {{ site.baseurl }}/assets/img/docs/getting-started-guide-exp/pass-pipeline-gitlab.png
-{%- endcapture -%}
-
-{% include two-up.html title="4. Congratulations 🎉" content=content imageURL=congrats-first-green-pipeline imageAlt="Passing pipeline" %}
-++++
---
-
-// step 2
-
-++++
-{% capture content %} You should see your pipeline start to run automatically—and pass! {% endcapture %}
+{% capture content %} Explore your passing pipeline and invite your teammates to join you, for free. By collaborating, you can troubleshoot, get pull requests approved, and build and test faster. {% endcapture %}
```

Comment:
Should we add a bit more instruction or a link to a page that shows them how to add team members?

Reply:
Yeah I think here it's implicit that you would look at the image and know to click the link on the dashboard but I agree we could make this clearer. It will be different for the different account types though

Reply:
Definitely and there are some upcoming changes too. I think it will be fine to just link out to another page either here or under "Next steps"